### ISSUE TITLE
Client stopped working overnight

### ISSUE THREAD

**Issue by Blubberislife** [gamejolt/issue-tracker#577]
It was working the night I started using it, (yesterday) but when I opened it up, it said that I hadn't installed any games- even though I have. I made sure to check the directory that the games were installed to, and it was the same directory that they are currently set to install. Additionally, it won't let me view any game's pages. It will show a progress bar at the top, but when it finishes, it stays on the same page.

tl;dr
Client suddenly stopped working, with no apparent explanation.

**Comment by YLivay** [comment]
Sounds like  two different issues.

Game pages unavailable suggests it was a problem with site. Maybe it hiccuped just when you checked?
Does it work now or does the issue persist?

As for the missing  games, sounds like something went wrong with your localdb. Scroll down to the footer and click "Send System Report".

**Comment by Blubberislife** [comment]
It started working again after I stopped the client's process in Task Manager. Thanks anyways, though!

-----Original Message-----
